code/modules/mod/mod_construction.dm

```diff
@@ -79,6 +79,20 @@
 /obj/item/mod/construction/plating/cosmohonk
 	theme = /datum/mod_theme/cosmohonk
 
+// Hardsuits as MOD plating
+
+/obj/item/clothing/suit/space/hardsuit/engine
+	theme = /datum/mod_theme/engineering
+
+/obj/item/clothing/suit/space/hardsuit/engine/atmos
+	theme = /datum/mod_theme/atmospheric
+
+/obj/item/clothing/suit/space/hardsuit/medical
+	theme = /datum/mod_theme/medical
+
+/obj/item/clothing/suit/space/hardsuit/security
+	theme = /datum/mod_theme/security
```

Comment:
Themes for every added hardsuit, please, even if they default to the civvie look or something.

Reply:
> Themes for every added hardsuit, please, even if they default to the civvie look or something.

Done. I've added it so that each hardsuit will upgrade into their respective MODsuit. This is going to be extremely funny when people realise this is a mechanic we have because the modsuits made this way have 0 modules on them or upgrades.

Reply:
> Wouldn't the modsuits made from plating have the same "problem"?

They do yeah